### ISSUE TITLE
Revive still worked on? Last update a year ago

### ISSUE THREAD

**Issue by Hoshi82** [LibreVR/Revive#1712]
As topic says, will there be any updates or is revive on hold?

**Comment by Rick6rimes** [comment]
I like to know that too

**Comment by mapotter99** [comment]
Yes, an update would be appreciated, especially with Lone Echo 2 right around the corner (hopefully!).

**Comment by Roencia** [comment]
Has development on Revive stopped?

**Comment by vvvnew** [comment]
> Yes, an update would be appreciated, especially with Lone Echo 2 right around the corner (hopefully!).

Lone echo 2 runs like crap on my G2. I think a revive update is needed
